string mesh_name
---
int8 success